msteams-platform/samples/virtual-assistant.md

```diff
@@ -541,12 +541,16 @@ Book-a-room bot is designed for private chats (personal/1:1 scope) only. Since w
     }
 ```
 
-You can also leverage existing skills from [Bot Framework Solutions repository](https://github.com/microsoft/botframework-solutions/tree/master/skills/csharp) or create a new skill altogether from scratch. Tutorials for the later can be found [here](https://microsoft.github.io/botframework-solutions/overview/skills/). Please refer to [documentation](https://docs.microsoft.com/azure/bot-service/skills-conceptual?view=azure-bot-service-4.0)   for Virtual Assistant and skills architecture.
+You can also leverage existing skills from [Bot Framework Solutions repository](https://github.com/microsoft/botframework-solutions/tree/master/skills/csharp) or create a new skill altogether from scratch. Tutorials for the later can be found [here](https://microsoft.github.io/botframework-solutions/overview/skills/). Please refer to [documentation](https://docs.microsoft.com/azure/bot-service/skills-conceptual?view=azure-bot-service-4.0&preserve-view=true) for Virtual Assistant and skills architecture.
+
+## Code sample
+
+| **Sample name** | **Description** | **C#** | **.NET** |
+|----------|-----------------|----------|------------------|
+| Updated visual studio template | Customized template to support teams capabilities. | [View](https://github.com/OfficeDev/microsoft-teams-apps-bookaroom/tree/nebhagat/microsoft-teams-apps-bookaroom-skill) |
+| Book-a-room bot skill code | Lets you quickly find and book a meeting room on the go. |  | [View](https://github.com/nebhagat/msteams-virtual-assistant-dotnet) |
```

Comment:
We should not include reference to personal repositories in documentation. Could you please get in touch with Nehal Bhagat and see if this sample is available in OfficeDev repo? Also, C# .NET should be single column.

Reply:
Unfortunately we don't have this repo available in OfficeDev org. @aosolis do you think we should move this repo? 
Till then please refer to my personal repo.

Reply:
@nebhagat Can you work with Wajeed to post this to the samples repo that he maintains?

Reply:
sure, will reach out to Wajeed